src/mineflayer/commands/party/Stream.mjs

```diff
@@ -1,11 +1,11 @@
 import { Permissions, WebhookMessageType } from "../../../utils/Interfaces.mjs";
 
 export default {
-  name: ["stream", "public", "open"], // This command will be triggered by either command1 or command2
-  ignore: false, // Whether to ignore this file or not
-  description:
-    "Opens the party to the public, or reopens it if an incompetant person transfers to themself without MVP++", // Description of the command
-  permission: Permissions.Trusted, // Permission level required to execute this command
+  name: ["open", "public", "stream"],
+  description: "Opens the party to the public",
+  usage: "!p open [size]",
```

Comment:
Maybe an indicator that size is optional/defaults to 100; also, should this mention that `open` can also re-open an existing/closed party?